mkdocs.yml

```
@@ -82,7 +82,9 @@ nav:
       - Kubernetes:
           - CLI:
               - Scanning: docs/kubernetes/cli/scanning.md
-              - Compliance: docs/kubernetes/cli/compliance.md
+              - Compliance: 
+                - NSA: docs/kubernetes/cli/nsa.md
```

I want to get the directory structure aligned with the document structure.

```suggestion
                - NSA: docs/kubernetes/cli/compliance/nsa.md
```